src/main/scala/exercises/06_Collections.scala

```diff
@@ -103,7 +106,10 @@ object Collections {
    * to implement in your own free-style way.
    */
   def elementExists[T](l: List[T], e: T): Boolean = {
-    error("fix me")
+    l match {
+      case List() => false
+      case x :: xs => (x == e) || elementExists(xs, e)
```

Comment:
So once it finds the element (and returns true), it stops, correct? Otherwise it keeps looking for the element in the first index of the remainder of the list?